js/custom.js

```diff
@@ -11,21 +11,21 @@ $("#menu-toggle").click(function(e) {
 });
 
 // Scrolls to the selected menu item on the page
-$(function() {
-  $('a[href*=#]:not([href=#])').click(function() {
-    if (location.pathname.replace(/^\//, '') == this.pathname.replace(/^\//, '') || location.hostname == this.hostname) {
+// $(function() {
```

Comment:
this method was preventing the carousel next and prev buttons from working correctly so I commented it out

Reply:
go ahead and delete the commented code, it came with the jekyll theme and won't be used.

Reply:
Actually, we do use that function once, in the "Let's go" button in the first section but go ahead and get rid of that too.

Reply:
sounds good